draft-ietf-quic-transport.md

```diff
@@ -515,8 +515,8 @@ A Version Negotiation packet has long headers with a type value of 0x01 and is
 sent only by servers.  The Version Negotiation packet is a response to a client
 packet that contains a version that is not supported by the server.
 
-The packet number, connection ID and version fields echo corresponding values
-from the triggering client packet.  This allows clients some assurance that the
+The connection ID and version fields echo corresponding values from the
+triggering client packet.  This allows clients some assurance that the
 server received the packet and that the Version Negotiation packet was not
 carried in a packet with a spoofed source address.
```

Comment:
I would add something like "The packet number field SHOULD be set to 0 and will be ignored by the receiver."

Reply:
I think it's fine not to check the packet number(as updated), but if so, then advising what to set there seems unnecessary?

Reply:
The sender has to put something in the packet number, so I think it's better to be explicit about what it should do.

Reply:
Note that if we decide to have a shorter packet header for Version Negotiation, a 0 is harmless. It advertises support for a version that cannot be supported, so it can be ignored.

I was going to suggest having explicit text about ignoring the packet number, but we might just remove it, so don't bother just yet.